.gitignore

```diff
@@ -11,3 +11,4 @@ __screenshots__
 /cypress/screenshots
 /.env*
 /reearth-config.json
+.idea/*
```

Comment:
Is this necessary for this PR? Or maybe connected with your onboarding session??

Reply:
it's because that I'm using WebStorm and no one else in the company is using it so I had to ignore it's config files